config.json

```diff
@@ -41,19 +41,13 @@
           "expand": 0
         },
         "items": [
-          {
-            "title": "Learning Labs",
-            "content": "https://developer.cisco.com/learning/modules/automating-cisco-defense-orchestrator/",
-            "type": "url"
-          },
           {
             "title": "Postman Collection",
-            "content": "https://github.com/cisco-lockhart/cdo-public-api-docs/blob/main/postman-collection.json",
-            "type": "url"
+            "content": "developer-resources/postman-collection.md"
```

Comment:
We need a proper file, otherwise pubhub has a bug whereby the Developer Resources tab does not do anything unless you click on the little grey arrow.